.github/workflows/dev-container.yml

```
@@ -186,7 +186,9 @@ jobs:
   behave:
     name: "Run: behave"
     runs-on: ubuntu-24.04
-    timeout-minutes: 20
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)